src/core/variable.jl

```diff
@@ -127,6 +128,68 @@ function variable_reactive_line_flow(pm::GenericPowerModel; bounded = true)
     return q
 end
 
+############## DC Lines ############################################
+function variable_line_flow_dc(pm::GenericPowerModel; kwargs...)
+    variable_active_line_flow_dc(pm; kwargs...)
+    variable_reactive_line_flow_dc(pm; kwargs...)

```

Comment:
Did this naming convention get updated?  The diff still looks like the old version.

If you are touching the names, I was also thinking maybe `_dcline_` would be more consistent than `_dc_line_`, your call.